### PR TITLE
feat(MRS): add a new resource to manage scaling policy

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
Running tool: /usr/local/go/bin/go test -timeout 9000000000s -run ^TestAccScalingPolicy_basic$ github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/mrs -v

=== RUN   TestAccScalingPolicy_basic
=== PAUSE TestAccScalingPolicy_basic
=== CONT  TestAccScalingPolicy_basic
--- PASS: TestAccScalingPolicy_basic (718.26s)
PASS
ok  	github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/mrs	719.021s

```

### CODE REVIEW

File: docs/resources/mapreduce_scaling_policy.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+* `uri` - (Required, String) Path of a custom automation script.  
+  Set this parameter to an OBS bucket path or a local VM path.  
+  OBS bucket path: Enter a script path manually. for example, s3a://XXX/scale.sh.  
+  Local VM path: Enter a script path. The script path must start with a slash (/) and end with .sh.
+
+* `parameters` - (Optional, String) Parameters of a custom automation script.  
+  Multiple parameters are separated by space.  
+  The following predefined system parameters can be transferred:  
+    + **${mrs_scale_node_num}**: Number of the nodes to be added or removed.
+    + **${mrs_scale_type}**: Scaling type. The value can be scale_out or scale_in.
```
Comment (houpeng80) [inline docs/resources/mapreduce_scaling_policy.md]:
```suggestion
    + **${mrs_scale_type}**: Scaling type. The value can be **scale_out** or **scale_in**.
```

File: docs/resources/mapreduce_scaling_policy.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+
+```bash
+$ terraform import huaweicloud_mapreduce_scaling_policy.test <cluster_id>/<node_group>
+```
+
+Note that the imported state may not be identical to your resource definition, due to some attributes missing from the
+API response, security or some other reason. The missing attributes include: `exec_scripts`.
+It is generally recommended running `terraform plan` after importing a cluster.
+You can then decide if changes should be applied to the cluster, or the resource definition
+should be updated to align with the resource. Also you can ignore changes as below.
```
Comment (houpeng80) [inline docs/resources/mapreduce_scaling_policy.md]:
```suggestion
It is generally recommended running `terraform plan` after importing a scaling policy.
You can then decide if changes should be applied to the scaling policy, or the resource definition
should be updated to align with the scaling policy. Also you can ignore changes as below.
```

File: huaweicloud/services/mrs/resource_huaweicloud_mapreduce_scaling_policy.go [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
```
Comment (houpeng80) [inline huaweicloud/services/mrs/resource_huaweicloud_mapreduce_scaling_policy.go]:
```suggestion
			"cluster_id": {
				Type:     schema.TypeString,
				Required: true,
				ForceNew: true,
				Description:` The MRS cluster ID to which the auto scaling policy applies.`,
			},
```

File: docs/resources/mapreduce_scaling_policy.md (hunk 2 above)
Comment (hmbig2) [reply docs/resources/mapreduce_scaling_policy.md]: done

File: docs/resources/mapreduce_scaling_policy.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/mapreduce_scaling_policy.md]: done

File: huaweicloud/services/mrs/resource_huaweicloud_mapreduce_scaling_policy.go (hunk 3 above)
Comment (hmbig2) [reply huaweicloud/services/mrs/resource_huaweicloud_mapreduce_scaling_policy.go]: done

**Pull request merged**